cli/tests/integration/test_tests.rs

```diff
@@ -19,6 +19,21 @@ fn no_color() {
   assert!(out.contains("test result: FAILED. 1 passed; 1 failed; 1 ignored; 0 measured; 0 filtered out"));
 }
 
+#[test]
+fn fail_fast_with_value() {
+  let (out, _) = util::run_and_collect_output(
+    false,
+    "test --fail-fast=2 test/fail_fast.ts",
+    None,
+    Some(vec![("NO_COLOR".to_owned(), "true".to_owned())]),
+    false,
+  );
+  // ANSI escape codes should be stripped.
+  assert!(out.contains("test test 1 ... FAILED"));
+  assert!(out.contains("test test 2 ... FAILED"));
+  assert!(out.contains("test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out"));
+}
```

Comment:
Please use `itest!` macro and add another file for testing - depending on already existing files is bad as there's a need to change you will have to change multiple test results